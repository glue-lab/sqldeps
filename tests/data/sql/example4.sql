-- Query with table alias, with and without database specification, and join, and where clauses
SELECT u.id, u.name, o.order_id
FROM my_db.users u
JOIN orders AS o ON u.id = o.user_id
WHERE u.status = 'active'
    AND o.order_date >= '2024-01-01'
    AND o.total_amount > 100.00
    AND u.email LIKE '%@company.com'
    AND o.order_type IN ('retail', 'wholesale')
    AND (
        o.shipping_status = 'pending'
        OR (o.shipping_status = 'processed' AND o.priority_level = 'high')
    );