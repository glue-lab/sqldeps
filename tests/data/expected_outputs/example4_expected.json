{
    "dependencies": {
        "my_db.users": ["email", "id", "name", "status"],
        "orders": ["order_date", "order_id", "order_type", "priority_level", "shipping_status", "total_amount", "user_id"]
    },
    "outputs": {
      
    }
}
